lib/appsignal/config.rb

```diff
@@ -147,6 +151,10 @@ def config_file
         root_path.nil? ? nil : File.join(root_path, 'config', 'appsignal.yml')
     end
 
+    def detect_from_system
+      self[:running_in_container] = true if Appsignal::System.container?
```

Comment:
Haven't looked at `Appsignal::System#container?`, but from the name it looks like it returns a boolean, so you'd be able to this:

``` ruby
 self[:running_in_container] = Appsignal::System.container?
```